okhttp/src/main/kotlin/okhttp3/internal/Util.kt

```diff
@@ -249,6 +249,14 @@ fun String.canParseAsIpAddress(): Boolean {
   return VERIFY_AS_IP_ADDRESS.matches(this)
 }
 
+/** Returns true if we should void putting this this header in an exception or toString(). */
+fun isSensitiveHeader(name: String): Boolean {
```

Comment:
Should this be a public API feature?

Reply:
I don’t think so. It’s the sort of thing you don’t know you need until it’s too late.